### ISSUE TITLE
Feature: Open Task Manager by double (or single) click on icon

### ISSUE THREAD

**Issue by leni8ec** [Kyome22/RunCat_for_windows#63]
It would be great to be able to open the **Task Manager** in Windows when the cat starts to run fast to see what happens.

**Comment by dim0147** [comment]
The number on the cat is the percent of CPU usage right?

**Comment by leni8ec** [comment]
No, I meant calling the task manager when you click on the running cat icon.

**Comment by tom-turbo** [comment]
Yes, that would be great!
Double Klicke on the cat - TaskManager opens

**Comment by Bowarc** [comment]
This feature is now live, this issue should be closed
